docs/gemstones/rpaste.md

```diff
@@ -0,0 +1,79 @@
+---
+title: rpaste - Pastebin Tool
+author: Steven Spencer
+contributors:
+tags:
+  - rpaste
+  - Mattermost
+  - pastebin
+---
+
+# Introduction to `rpaste`
+
+`rpaste` is a tool for sharing code, log output, and other extra long text. It is a pastebin created by the Rocky Linux developers. It is particularly useful when you need to share something publicly, but do not want to dominate the feed with your text. This is extremely important when using Mattermost, which has bridges to other IRC services. It can be installed on any Rocky Linux system. If your desktop machine isn't Rocky Linux or if you simply do not want to install the tool, you can use it manually, by accessing the [pinnwand URL](https://rpa.st)and then paste the system output or text that you want to share. `rpaste` allows you to create this information automatically.
+
+## Installation
+
+Installation of `rpaste` on Rocky Linux:
+
+```
+sudo dnf --enablerepo=extras install rpaste
+```
+
+## Uses
+
+For major system problems, you may need to send all of your system's information so that it can be reviewed for problems. To do this run:
+
+```
+rpaste --sysinfo
+```
+
+Which, will return the link to the pinnwand page:
+
+```bash
+Uploading...
+Paste URL:   https://rpa.st/2GIQ
+Raw URL:     https://rpa.st/raw/2GIQ
+Removal URL: https://rpa.st/remove/YBWRFULDFCGTTJ4ASNLQ6UAQTA
+```
+
+You can then review the information yourself in a browser and decide if you want to keep it or remove it and start over. If you want to keep it, you can copy the "Paste URL" and share that with whomever you are working with, or in the feed on Mattermost. To remove, simply copy the "Removal URL" and open that in your browser.
+
+You can add content to your pastebin by piping the content. As an example, if you wanted to add content from your `/var/log/messages` file from March 10, you might do this:
+
+```bash
+sudo more /var/log/messages | grep 'Mar 10' | rpaste
+```
+
+## `rpaste` help
+
+To get help with the command, simply type:
+
+```
+rpaste --help
+```
+
+Which gives the following output:
+
+```bash
+rpaste: A paste utility originally made for the Rocky paste service
+
+Usage: rpaste [options] [filepath]
+       command | rpaste [options]
+
+This command can take a file or standard in as input
+
+Options:
+--life value, -x value      Sets the life time of a paste (1hour, 1day, 1week) (default: 1hour)
```

Comment:
life time = lifetime

Reply:
You are right, but again, this is the exact wording that is returned by `rpaste --help` so I left it in.